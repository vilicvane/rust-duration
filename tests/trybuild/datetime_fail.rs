fn main() {
  lits::datetime!("not-a-datetime");
}
