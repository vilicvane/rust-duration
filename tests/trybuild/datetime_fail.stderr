error: failed to parse string as SystemTime: timestamp format is invalid
 --> tests/trybuild/datetime_fail.rs:2:19
  |
2 |   lits::datetime!("not-a-datetime");
  |                   ^^^^^^^^^^^^^^^^
